### ISSUE TITLE
Add project showcase to landing page

### ISSUE THREAD

**Issue by gavin-k-lee** [SwissDataScienceCenter/renku-ui#1932]
Implement carousel for project showcase on landing page.

Depends on #537 and #538

Acceptance criteria:
- [ ] Landing page shows project showcase

**Comment by lokijuhy** [comment]
Sekhar to show what the current built in template is for project showcases, and see if it needs any modification.

**Comment by ciyer** [comment]
We have the start of some infrastructure for this, but it needs to be completed to support this. I propose showing a few thumbnails with descriptions that link to Renku projects that contain details. Instead of using thumbnails, we could do a carousel, as suggested above -- this would have little impact to the implementation, it would just mean that there is more space available for the image and description, since the carousel would take up the entire area of the section.

# Mock-up

![Image](https://user-images.githubusercontent.com/1196411/179229206-a8bd4e90-df41-4aec-8e48-65fd37ce3f9a.png)

# Implementation

The showcase section would be configured per deployment and be optional. The values file already contains a section for this https://github.com/SwissDataScienceCenter/renku-ui/blob/master/helm-chart/renku-ui/values.yaml#L160.

For the content of the projects, I propose an array of objects with 4 fields: `title`, `description` (markdown), `imageUrl`, `projectUrl`. So,

```
    projects:
      - 
        title: _project title_
        description: _markdown_
        imageUrl: _url_
        projectUrl: _url_
```

Does this work for you?

**Comment by gavin-k-lee** [comment]
That looks great! The `projectUrl` could in theory be any project, right? We might put a GitLab group there to be able to link to a list of relevant repositories.

**Comment by ciyer** [comment]
Yes, in fact it could be a link to anything, for example a readme or notebook within a repository.

**Comment by gavin-k-lee** [comment]
Note: since this involves external collaborators, I'd like to reach out to them and show them this in a test deployment before merging/releasing.

**Comment by gavin-k-lee** [comment]
We have 2 Renku GitLab groups we'd like to show on the carousel: 
- Title: Learn Renku
- Description: Check out our in-depth learning paths on Renku. 
- ImageUrl: A Renku logo should suffice here - any other ideas?
- ProjectUrl: https://renkulab.io/gitlab/learn-renku (TBC: should we link the Renku one? We need to display the group uniquely.)

- Title: Renku Gallery
- Description: A showcase of computational tools brought to Renku.
- ImageUrl: The GitLab group icon looks fine.
- ProjectUrl: https://renkulab.io/gitlab/renku-stories (TBC: should we link the Renku one? We need to display the group uniquely.)

**Comment by gavin-k-lee** [comment]
With the 4 objects to show here, each of them has slightly different dimensions for their image and different description lengths. We should develop a design for this in a wireframe to better understand and visualise the spacing/impact of each of the elements.

**Comment by lokijuhy** [comment]
@gavin-k-lee Re Learn-Renku gitlab group - that sends people to GitLab, which doesn't make a lot of sense. It should take the user to Renku- maybe a wrapper project that describes and links to all content in Learn-Renku?

**Comment by lokijuhy** [comment]
Design issues to be resolved:
* Need some additional text under "See Renku in action"
* Decide between 1-item carousel (more room for description) or multi-item static grid (less room, but better overview)
  -> Laura to Ask Oksana if we can get Coteries time for this

**Comment by gavin-k-lee** [comment]
@lokijuhy re gitlab groups - agree that sending them to GitLab is not optimal. However the current (and I believe also the cross-entity) search does not allow specifying the actual group. For example we could link to this

![Image](https://user-images.githubusercontent.com/22785147/186082468-d9a79290-551a-4abc-ad3d-b53ca8d7e42a.png)

but the group we want (Renku Gallery / renku-stories) cannot be specified to be selected by default. It might be confusing to users to show just the cell-post/napari projects. 

I was originally thinking of tagging all the relevant projects with a unique tag but we also cannot search based on keywords currently - though in the cross-entity search one will be able to, e.g. https://renkulab.io/knowledge-graph/entities?query=dataviz.
